api/collections.js

```diff
@@ -135,7 +135,7 @@ function updateMyQuoteCollection(req, res, next) {
   const immutables = ['owner', 'quotes'];
   const update = dropProperties(req.body, immutables);
   const query = { owner: req.user.id, _id: req.params.id };
-  QuoteCollection.findOneAndUpdate(query, update, { new: true })
+  QuoteCollection.findOneAndUpdate(query, update, { new: true, runValidators: true })
```

Comment:
Well, if you extracted everything into local vars do that for options too :bowtie: